src/Tizen.NUI.Components/Controls/Pagination.cs

```diff
@@ -366,6 +366,10 @@ public int SelectedIndex
                 selectedIndex = refinedValue;
 
                 SelectIn(indicatorList[selectedIndex]);
+
+                if (IsHighlighted) {
```

Comment:
Please update it like the below.
```suggestion
                if (IsHighlighted)
                {
```

Reply:
Done